src/main/java/org/codeforamerica/messaging/models/Message.java

```diff
@@ -38,4 +38,16 @@ public class Message {
     private OffsetDateTime creationTimestamp;
     @UpdateTimestamp
     private OffsetDateTime updateTimestamp;
+
+    public boolean needToSendEmail() {
+        return toEmail != null && emailMessage == null;
+    }
+
+    public boolean needToSendSms() {
+        return toPhone != null && smsMessage == null;
+    }
+
+    public String getStatus() {
+        return needToSendEmail() || needToSendSms()? "pending": "completed";
```

Comment:
Should we add this status check to the `getMessage` endpoint?

Also, tiny nit that I would usually add spaces for ternaries like so
```suggestion
        return needToSendEmail() || needToSendSms() ? "pending" : "completed";
```

Reply:
Good call, I'll add the tests.